backend/src/main/java/com/zpi/backend/reservations/Reservation.java

```diff
@@ -59,12 +59,17 @@ public Reservation fromDTO(NewReservationDTO newReservationDTO, User renter, Gam
         this.gameInstance = gameInstance;
         this.startDate = newReservationDTO.getStartDate();
         this.endDate = newReservationDTO.getEndDate();
-        this.renterComment = newReservationDTO.getRenterComment();
+        if(this.renterComment !=null)
+            this.renterComment = newReservationDTO.getRenterComment().orElse(null);
         this.timestamp = new Date(System.currentTimeMillis());
         this.duration = DateUtils.getDuration(startDate,endDate);
         this.reservationId = UUID.randomUUID().toString();
         return this;
     }
+
+    public int getDuration() {
+        return DateUtils.getDuration(startDate,endDate);
```

Comment:
Duration does not take start date and end date in duration (always is 2 days less than it should be)

Reply:
*1 day less